build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/GenerateSupportedPlatforms.cs

```diff
@@ -86,5 +82,22 @@ public override bool Execute ()
 
 			return !Log.HasLoggedErrors;
 		}
+
+		static AndroidVersion ToVersion (ITaskItem item)
+		{
+			/*
+			<AndroidApiInfo Include="v12.0.99">
+				<Name>Sv2</Name>
+				<Level>32</Level>
+				<Id>Sv2</Id>
+				<Stable>False</Stable>
+			</AndroidApiInfo>
+			*/
+
+			int.TryParse (item.GetMetadata ("Level"), out int apiLevel);
```

Comment:
Shouldn't we check the return value here?  Sure, this is an "internal" file, but if someone hand-edits it *anyway*, the build could get perhaps "weird"?

Reply:
The task does `.Where (v => v.ApiLevel >= MinimumApiLevel)`, so it should skip any 0's.